docs/reference/Admin_Actions.md

```diff
@@ -4,6 +4,52 @@ When you consider that most backend services are a black box of code and decisio
 
 With that said this document will serve as the ongoing history of administrative actions that must be taken against the backend.
 
+## 2023 - August 16
+
+### appcelerator-titanium && titanium
+
+When the developers of `appcelerator-titanium` wanted to rebrand their package to `titanium` it seemed the backend was encountering errors attempting to do so. The developers then went ahead to create a brand new package `titanium` and continue development there. Unfortunately, also encountering errors when attempting to delete the now defunk package `appcelerator-titanium`. The developers had then [reached out](https://github.com/pulsar-edit/package-backend/issues/190) to the Pulsar Backend team for assistance.
+
+Since none of this issues where the fault of the developers and seem to fully fall on the backend itself, the request was happy to be met, which attempted to do the following:
```

Comment:
```suggestion
Since none of these issues were the fault of the developers and seem to fully fall on the backend itself, the request was happy to be met, which attempted to do the following:
```